views/dataset/summary.templ

```diff
@@ -0,0 +1,204 @@
+package dataset
+
+import (
+    "github.com/ugent-library/biblio-backoffice/ctx"
+    "github.com/ugent-library/biblio-backoffice/models"
+    contributorviews "github.com/ugent-library/biblio-backoffice/views/contributor"
+    relatedorganizationviews "github.com/ugent-library/biblio-backoffice/views/relatedorganization"
+    urlviews "github.com/ugent-library/biblio-backoffice/views/url"
+    "github.com/ugent-library/biblio-backoffice/views"
+    "net/url"
+    "fmt"
+)
+
+type SummaryArgs struct {
+    Dataset *models.Dataset
+    URL *url.URL
+    Actions templ.Component
+    Footer templ.Component
+    Links templ.Component
+}
+ 
+func getFirstDatasetIdentifier(dataset *models.Dataset) (string, []string) {
+    for typ, vals := range dataset.Identifiers {
+        return typ, vals
+    }
+    return "", nil
+}
+
+templ Summary(c *ctx.Ctx, args SummaryArgs) {
```

Comment:
@verheyenkoen @nicolasfranck just a general remark: we'll have to decide on an args style
- put everything  in an Args struct
- put everything except the ctx in an Args struct
- put everything except ctx and required arguments in an Opts struct
What are your thoughts?
Not a blocker, these things are easily refactored.

Reply:
Since an args struct is a kinda-sorta solution to optional arguments I was thinking the last one but you'd still need to pass on nil for the Opts struct parameter if you don't want to pass any. I'd leave context separate anyway as the first argument.

Reply:
* ctx.Ctx should always be the first argument. Even if we could embed it into a required Args. A bit like the context.Context that is used everywhere as the first argument. Lets not forget that for simpler cases with not much arguments an Args struct might be overkill, and in those cases we still have to supply the ctx. That way it is the same for the style with an Args struct. (`Method(c ctx.Ctx, args Args)`) or with separate arguments (`Method(c ctx.Ctx, arg1, arg2 ..)`)
* I'm not convinced that optional arguments should be all be in an optional variadic arguments like those Opts. These are arguments to templates, struct attributes are never required by the language, and what if there more than one optional types of arguments?

Reply:
@nicolasfranck not variadic, just a struct as last argument
ex. `ShowPublication(ctx, pub, optionalStuff{})`

Reply:
but we have consensus on leaving ctx out of the args/opts